src/Beerware.xml

```diff
@@ -0,0 +1,12 @@
+<SPDX name="Beerware License" identifier="Beerware" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing/Beerware</url>
```

Comment:
I'd like to add another url here... (but that would be a fork of the current list...)
https://people.freebsd.org/~phk/